x-pack/plugins/canvas/init.js

```diff
@@ -12,16 +12,18 @@ import { registerCanvasUsageCollector } from './server/usage';
 import { loadSampleData } from './server/sample_data';
 
 export default async function(server /*options*/) {
-  server.injectUiAppVars('canvas', () => {
+  server.injectUiAppVars('canvas', async () => {
```

Comment:
@w33ble does this look good to you? thx!

Reply:
Yup, made a build, the values were what they should be, and everything seemed to work.